docs/configuration.rst

```diff
@@ -102,6 +102,9 @@ support or e.g. mismatching TLS versions to deliver emails to Mailu.
 
 .. _fetchmail:
 
+When ``FETCHMAIL_ENABLED`` is set to ``True``, the fetchmail functionality is shown in the admin interface.
```

Comment:
Maybe this should be changed from "is shown" to "is enabled", as you'll get a 404 for all calls.